podman/containers/__init__.py

```diff
@@ -0,0 +1,315 @@
+"""images provides the operations against images for a Podman service."""
+import json
+import logging
+
+import podman.errors as errors
+
+
+def make_call(api, endpoint, method='GET', params=None, body=None):
+    """A helper function to keep things DRY"""
+    path = api.join(endpoint, params)
+    print(path)
```

Comment:
should drop this print

Reply:
whoops, rookie move there.  this has been removed.